dev/metrics/README.md

```diff
@@ -0,0 +1,31 @@
+# Readme
+This directory contains some docker files to help you set up metrics tracking on an alluxio cluster. The scripts uses opentelemetry's auto instrumentation java agent to generate tracing information for GRPC and S3 calls, which is visualized using Jaeger and Prometheus. 
+
```

Comment:
do you have any references/links that you used to figure out all this, especially for the jaeger setup with all the prometheus stuff hooked up?